screensaver.bing/resources/skins/default/skin.xml

```diff
@@ -0,0 +1,10 @@
+<skin>
```

Comment:
I don't think this is of any use in addons. I'd consider removing the file

Reply:
I will, if we can leave for now I'll remove asap.

Reply:
removed...